src/main/java/emu/grasscutter/game/player/Player.java

```diff
@@ -753,7 +754,14 @@ public void addShopLimit(int goodsId, int boughtCount, int nextRefreshTime) {
 		}
 		this.save();
 	}
-
+	public boolean getstamina() {
```

Comment:
Please use proper camelcase naming convention. (Ex. getstamina -> getStamina) Other than that, everything looks good.

Reply:
Done.Thanks.